user-service/src/main/java/org/java/mentorship/user/controller/UserController.java

```diff
@@ -18,4 +19,13 @@ public ResponseEntity<List<User>> getUsers() {
 
         return ResponseEntity.ok(Collections.singletonList(user));
     }
+
+    @GetMapping("/users/{id}")
+    public ResponseEntity<User> getUserById(@PathVariable(name="id") Integer id) {
```

Comment:
Can we create the UserService and UserRepository as well?
You can run the user-service from IntelliJ until we've got the database in Docker.

Reply:
Just done that, UserRepository has a temporary List<UserEntity> field until database is created